### PR TITLE
Update CentOS and Ubuntu image versions to 7.6 and 18.04 

### PR DESCRIPTION
**Pull request by brtkwr**

### CODE REVIEW

File: etc/p3-config/p3-config.yml [hunk 1]
```diff
@@ -463,7 +463,7 @@ p3_image_fedora_atomic_27:
 
 # Latest CentOS image
 p3_image_centos:
-  name: "CentOS7"
+  name: "CentOS7.6"
```
Comment (markgoddard) [inline etc/p3-config/p3-config.yml]: If we rename the image it could break clusters that depend on it

**Review by markgoddard — CHANGES_REQUESTED** [review]

File: etc/p3-config/p3-config.yml [hunk 2]
```diff
@@ -603,8 +603,7 @@ p3_image_mlnx_ofed:
     - "vm"
     - "systemd-modules-load"
   env:
-    DIB_MLNX_OFED_VERSION: "4.1-1"
-    DIB_MLNX_OFED_REPO: "{{ p3_repo_server_url_mlnx_ofed }}"
+    DIB_MLNX_OFED_REPO: "https://linux.mellanox.com/public/repo/mlnx_ofed/latest/rhel7.6/mellanox_mlnx_ofed.repo"
```
Comment (markgoddard) [inline etc/p3-config/p3-config.yml]: Don't you want https://linux.mellanox.com/public/repo/mlnx_ofed/latest/rhel7.6/x86_64/?

File: etc/p3-config/p3-config.yml [hunk 3]
```diff
@@ -603,8 +603,7 @@ p3_image_mlnx_ofed:
     - "vm"
     - "systemd-modules-load"
   env:
-    DIB_MLNX_OFED_VERSION: "4.1-1"
```
Comment (markgoddard) [inline etc/p3-config/p3-config.yml:606]: export MLNX_OFED_VERSION=${DIB_MLNX_OFED_VERSION:?"A revision of OFED is required"}

File: etc/p3-config/p3-config.yml (hunk 2 above)
Comment (brtkwr) [reply etc/p3-config/p3-config.yml]: No

File: etc/p3-config/p3-config.yml (hunk 3 above)
Comment (brtkwr) [reply etc/p3-config/p3-config.yml:606]: Now, if MLNX_OFED_REPO_FILE is specified, this is overridden.

**Review by markgoddard — APPROVED** [review]

**Comment by brtkwr** [comment]
Thanks, only you have the power to merge here

**Pull request merged**